### DISCUSSION TITLE
How to know the underlying tests is xunit or nunit?

### DISCUSSION THREAD

**Discussion by kabdul-incomm** [xunit/xunit#2830] [Question] [answered]
We have two projects xunit and nunit which uses the same Page object Model (POM ) and in POM we refer extent reports to add some information about the tests and we do this using Testcontext of Nunit and it errors out for xunit. Here, I want to know in my POM project "How can I know whether the underlying tests is xunit or nunit"?

**Comment by bradwilson** [comment, marked as answer]
I'm not aware of anything that can do this automatically. At runtime, you could look at the currently loaded assemblies (with `AppDomain.CurrentDomain.GetAssemblies()`) and look for anything that starts with `nunit` or `xunit` as a way to guess.
